helm/prometheus-rules/templates/alerting-rules/docker.rules.yml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.managementCluster.provider.flavor "vintage" }}
```

Comment:
I'm guessing the reason here is that CAPI uses containerd instead of docker?
Maybe we could add a comment so we understand it at first sight in the future: 
```suggestion
# newer clusters don't use docker anymore
{{- if eq .Values.managementCluster.provider.flavor "vintage" }}
```

Reply:
it's not capi, old vintage clusters support that as well